cpp/src/parquet/properties.h

```diff
@@ -1180,4 +1180,10 @@ struct ArrowWriteContext {
 PARQUET_EXPORT
 std::shared_ptr<ArrowWriterProperties> default_arrow_writer_properties();
 
+// PLAIN_DICTIONARY is deprecated but used to be used as a dictionary index
```

Comment:
Why put this in `properties.h`? Perhaps `types.h` would be more appropriate.

Reply:
Aha, previously `dictionary_index_encoding()` is in `properties.h`. I'll move it

Reply:
Moved to `types.h` now